### PR TITLE
Basic tf structure

### PR DESCRIPTION
**Pull request by GuruduGanesh**
Please review the initial structure.

### CODE REVIEW

**Comment by dhayanands** [comment]
What is the going to be difference between security, network & route?

should the network & route be part of the same module?

File: modules/azurerm_route_table/outputs.tf
Comment (meejurusiva) [inline modules/azurerm_route_table/outputs.tf:1]: Is this need to be removed ? Can you check.

**Review by meejurusiva — CHANGES_REQUESTED** [review]
Are we going to use existing codes in  azure modules or are those for references ?

File: modules/azurerm_route_table/outputs.tf
Comment (GuruduGanesh) [reply modules/azurerm_route_table/outputs.tf:1]: Yes, It Microsoft provided module. We need to use it in our code.
Source of modules is - https://github.com/sysco-middleware/terraform-azure-modules

**Review by dhayanands — APPROVED** [review]
for now we can start with this code

**Review by meejurusiva — APPROVED** [review]

**Pull request merged**